lib/Loop/Loop.php

```diff
@@ -302,7 +302,7 @@ protected function runStreams(?float $timeout): void
     /**
      * A list of timers, added by setTimeout.
      *
-     * @var array<int, mixed>
+     * @var array<int, array{0:float, 1:callable}>
```

Comment:
we might even define the callable signatures which can help to identify issues at the caller side (can be a separate PR)